clang/include/clang/AST/Type.h

```diff
@@ -6162,6 +6163,63 @@ class PipeType : public Type, public llvm::FoldingSetNode {
   bool isReadOnly() const { return isRead; }
 };
 
+// A fixed int type of a specified bitwidth.
```

Comment:
I feel like it's possible to define only `DependentExtIntType `class here. But it will come in cost of extra `isInstantiationDependent()` in the code.